src/renderers/native/findNodeHandle.js

```diff
@@ -61,10 +60,10 @@ import type {ReactInstance} from 'ReactInstanceType';
 function findNodeHandle(componentOrHandle: any): any {
   if (__DEV__) {
     // TODO: fix this unsafe cast to work with Fiber.
```

Comment:
For now, I'm copying what we do in [`findDOMNode`](https://github.com/facebook/react/blob/6ab2869a20e5e9a6f1f0ac02da34692dac9c1c9d/src/renderers/dom/shared/findDOMNode.js#L35) but I left this TODO in place.